plugins/source/gcp/README.md

```diff
@@ -0,0 +1,73 @@
+# GCP Source Plugin
+
+GCP Source plugin for CloudQuery extract configuration from varietly of GCP APIs and loads them to any of the supported CloudQuery Destination plugins.
+
+## Links
+
+- Tables: [./docs/tables/README.md](./docs/tables/README.md)
+- Homepage: [https://cloudquery.io](https://cloudquery.io)
+- Documentation: [https://docs.cloudquery.io](https://docs.cloudquery.io)
+- Discord: [https://cloudquery.io/discord](https://cloudquery.io/discord)
```

Comment:
```suggestion
- [Tables](./docs/tables/README.md)
- [Homepage](https://www.cloudquery.io)
- [Documentation](https://www.cloudquery.io/docs)
- [Discord](https://www.cloudquery.io/discord)
```

Might be nicer not to have file paths as link names.
This is how it looks before the suggestion
![image](https://user-images.githubusercontent.com/26760571/191711108-243b1291-820d-49d7-b962-6227e049c18c.png)

Reply:
I agree on the links but don't agree on the `www` as without it those are the canonical links

Reply:
Fixed the links